packages/cacti-plugin-copm-corda/sample/src/main/kotlin/com/copmCorda/interop/corda/RemoteCopmContractCordaFungible.kt

```diff
@@ -0,0 +1,18 @@
+package com.copmCorda.interop.corda
+
+import org.hyperledger.cacti.plugin.copm.interop.RemoteCopmContract
+import org.hyperledger.cacti.plugin.copm.types.DLAsset
+import org.hyperledger.cacti.plugin.copm.types.DLTransactionParams
+import org.hyperledger.cacti.plugin.copm.validators.ValidatedClaimPledgedAssetV1Request
+
+class RemoteCopmContractCordaFungible : RemoteCopmContract {
+    override fun matchesNetworkTypeAndAsset(networkType: String, asset: DLAsset): Boolean {
+        return networkType == "corda" && ! asset.isNFT
+    }
+
+    override fun getPledgeInfoCmd(claim: ValidatedClaimPledgedAssetV1Request): DLTransactionParams {
+        return DLTransactionParams("com.cordaSimpleApplication.flow",
```

Comment:
hardcoded value here. I'll suggest take it from parameters, or configurations.

Reply:
Sorry I just saw that this is inside `sample` directory, so for every application someone needs to write code similar to these files?

Reply:
Correct.  This is left up to the app for now (as opposed to json files) as in larger applications configuration info could be stored in, for instance, a database to make it easier to administer.  

As I have mentioned elsewhere, I also think some of this application configuration could be pushed down to the contracts.  As requested by @VRamakrishna , have a working draft where asset type registration is done in the interop contract: https://github.com/jenniferlianne/cacti/tree/copm_register_assets_in_chaincode

Reply:
okay makes sense.